js/iframeResizer.js

```diff
@@ -201,7 +201,7 @@
 				}
 
 				function checkSingle(){
-					var remoteHost  = settings[iframeId].remoteHost;
+					var remoteHost  = settings[iframeId] && settings[iframeId].remoteHost;
```

Comment:
Nope, in JavaScript the `&&` operator returns the last value or the first non-truthy value. Essentially this simply protects from accessing `.remoteHost` when `settings[iframeId]` is undefined. So, when `settings[iframeId]` is undefined, `remoteHost` is assigned undefined; and when `settings[iframeId]` is an object, `remoteHost` is assigned the value `settings[iframeId].remoteHost`.